globals.go

```diff
@@ -468,6 +468,8 @@ type ModificationAction string
 const (
 	PackageName = "github.com/evergreen-ci/evergreen"
 
+	OtelAttributeMaxLength = 10000
```

Comment:
This is Honeycomb's limit, right?

Reply:
This limit is self-imposed (HC doesn't care as long as the whole event is under 100kb). 
In https://github.com/evergreen-ci/evergreen/pull/6769 we limited the string length to prevent batches from being over the gRPC message size limit and it was determined that this limit should be enough for p99 of the `db.statement` strings.

Reply:
Okay, good. It's better that we code to our _own_ limits, and not ones that Honeycomb imposes. That helps us keep flexible in case we want to move OTel providers in the future.